docsbox/docs/tasks.py

```diff
@@ -12,26 +12,42 @@
 from docsbox.docs.via_controller import *
 
 
-def get_task(task_id):
+def get_task(task_id: str):
     queue = rq.get_queue()
     return queue.fetch_job(task_id)
 
+def remove_file(path: str):
+    os.remove(path)
 
-def remove_file(path):
-    return os.remove(path)
+class FileInfo:
```

Comment:
Placing these all around the existing files doesn't feel right. Maybe create a new folder for these and a new file for each class.

Reply:
Done